packages/php-wasm/universal/src/lib/logger.ts

```diff
@@ -0,0 +1,121 @@
+import { UniversalPHP } from "./universal-php";
+
+export enum LogSeverity {
+    // A dine grained debugging event. Typically disabled in default configurations.
+    Trace = 1,
+    // A debugging event.
+    Debug = 5,
+    // An informational event. Indicates that an event happened.
+    Info = 9,
+    // A warning event. Not an error but is likely more important than an informational event.
+    Warn = 13,
+    // An error event. Something went wrong.
+    Error = 17,
+    // A fatal error such as application or system crash.
+    Fatal = 21,
+}
+
+export type Log = {
+    // Time when the event occurred.
+    timestamp: number;
+    // Time when the event was observed.
+    observedTimestamp: number;
+    // Request trace id.
+    traceId: number;
+    // Request span id.
+    spanId: number; // Not implemented
```

Comment:
Do these parameters come from OpenTelemetry spec? If so, let's document that.

Reply:
Yes, I added a comment about it.
Also after chatting with @dmsnell, I'm not sure if using OpenTelemetry standards makes sense, or if we should send everything to the debug.log in the PHP standard.